images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -1399,26 +1380,7 @@ set -x
 
 # if we have LAGOON_POSTROLLOUT_DISABLED set, don't try to run any pre-rollout tasks
 if [ "${LAGOON_POSTROLLOUT_DISABLED}" != "true" ]; then
```

Comment:
as above

Reply:
This needs to be done in the build-deploy-tool first, closing this for future PR